crates/core/tedge_api/src/messages.rs

```diff
@@ -27,12 +31,12 @@ where
         serde_json::from_slice(bytes)
     }
 
-    fn to_json(&self) -> Result<String, serde_json::Error> {
-        serde_json::to_string(self)
+    fn to_json(&self) -> String {
+        serde_json::to_string(self).unwrap() // all thin-edge data can be serialize to json
```

Comment:
Looked bold, but makes sense.

Reply:
I decided to move that way when I saw the impact for nothing on so many places: https://github.com/thin-edge/thin-edge.io/pull/2245/commits/162a83ba520ccd4862874b1d7eed7b0d89cc4a23.